app/views/hyrax/admin/admin_sets/_form_visibility.html.erb

```diff
@@ -44,7 +44,7 @@
                   <h3><%= t('.visibility.title') %></h3>
                   <p><%= t('.visibility.description') %></p>
                   <%# List each option in a <div class='radio'> tag %>
-                  <%= f.collection_radio_buttons :visibility, f.object.visibility_options, :first, :last, item_wrapper_tag: :div, item_wrapper_class: 'radio' do |b| %>
+                  <%= f.collection_radio_buttons :visibility, f.object.visibility_options, :first, :last, item_wrapper_tag: :div, item_wrapper_class: 'radio', checked: f.object[:visibility] do |b| %>
```

Comment:
The reason this isn't working is because Tag::Base gets the value by sending `:visibility` to the form object (https://github.com/rails/rails/blob/92703a9ea5d8b96f30e0b706b801c9185ef14f0e/actionview/lib/action_view/helpers/tags/base.rb#L28)  this visibility method has been used by this method: https://github.com/projecthydra-labs/hyrax/blob/91729ff3bf48e1d7be1084bbe418d0c393ce46a2/app/forms/hyrax/forms/permission_template_form.rb#L15. So you probably want to rename that method.  Then make a "visibility" method that just returns the correct value.